### ISSUE TITLE
Mention PyDrive uses v2 of Drive API

### ISSUE THREAD

**Issue by mlovic** [googledrive/PyDrive#99]
I think it would be helpful to mention in the docs/README, that this library uses Drive API v2, not v3 which has been around for a while. It took me a while to realize this as I kept getting "Invalid Query" errors for using "name" instead of "title".

**Comment by RNabel** [comment]
Good point - thank you for opening the issue. I changed the README in the development branch and will close this as soon as the next release is published.

**Comment by Lykos153** [comment]
Are there any plans to migrate to v3?

**Comment by RNabel** [comment]
There are no active plans to move to v3. @sqrrrl had an open PR some time ago, though. He may be willing to have another stab at it.

**Comment by rutgerhofste** [comment]
Just wasted 1 hour writing queries using V3 API.
